index.sh

```diff
@@ -1,11 +1,4 @@
-s=$(which mighty-batch)
-if [ "$s" = "" ];
-then
-    npm install -g mighty-batch
-    cd tools
-    npm install
-    cd ..    
-fi
-tar -zxf outdoors_posts.tar.gz
-mighty-batch --threads 1 --workers 2 --host localhost --json outdoors_posts.json --property Paragraphs
-node tools/load.js --files vectors/outdoors_posts.json/ --name outdoors
+#!/usr/bin/env bash
+
+docker build ./tools --tag=mighty-indexer
```

Comment:
This command will build the image only once. Second and further launches will re-use cache, so it will be fast.